lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -72,6 +72,12 @@ static Type lowerType(Type type) {
     }
     return hw::StructType::get(type.getContext(), hwfields);
   }
+  if (FVectorType vec = firType.dyn_cast<FVectorType>()) {
```

Comment:
You can land this addition directly if you want.  Or as part of this, doesn't matter.